### PR TITLE
Replace laracasts/TestDummy with Laravel Factories

### PR DESCRIPTION
**Pull request by tomdiggle**
This PR address issue #231 by updating the database seeders and tests to use the laravel way to prepare factories.

### CODE REVIEW

File: tests/AccountTest.php [hunk 1]
```diff
@@ -33,27 +31,25 @@ function invalid_signups_dont_proceed()
             ->see('The password field is required')
             ->see('The email field is required');
 
-        $this->assertEquals(0, User::all()->count());
+        $this->assertEquals(0, App\User::all()->count());
```
Comment (mattstauffer) [inline tests/AccountTest.php]: Can you move App\User back into the import block?

File: tests/AccountTest.php [hunk 2]
```diff
@@ -33,27 +31,25 @@ function invalid_signups_dont_proceed()
...
     }
 
     /** @test */
     function users_can_log_in()
     {
-        $user = Factory::create('user', [
-            'password' => bcrypt('super-secret'),
-        ]);
+        $user = factory(App\User::class)->create();
```
Comment (mattstauffer) [inline tests/AccountTest.php]: Can we keep it where it's manually passing the password like previously so it's more obvious the connection?

File: tests/AccountTest.php (hunk 1 above)
Comment (mattstauffer) [reply tests/AccountTest.php]: Same for all the other classes in these tests.

**Comment by tomdiggle** [comment]
Do you want me to add the classes back into the import block for all tests like commit bc0616816114f5e143fd02d61c657de3577ff9d4 @mattstauffer?

**Comment by mattstauffer** [comment]
@tomdiggle that's fine just for this one, thank you! Is this ready for one final review before we merge? Thank you!!

**Comment by tomdiggle** [comment]
I'm happy with it if you are 👍

**Comment by mattstauffer** [comment]
WOO! Thanks @tomdiggle !

**Pull request merged**